packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -371,8 +371,8 @@ const ProductList = (props) => {
             {...rest}
         >
             <Helmet>
-                <title>{category?.pageTitle}</title>
-                <meta name="description" content={category?.pageDescription} />
+                <title>{category?.pageTitle ?? searchQuery}</title>
+                <meta name="description" content={category?.pageDescription ?? searchQuery} />
```

Comment:
SEO lighthouse check was failing since PLP pages with search queries (non category) had no title/description in header